### PR TITLE
#3341 Vaccine Placeholders

### LINKED ISSUE #3341: Vaccine Module page template and structure
## Is your feature request related to a problem? Please describe.

Adding a few pages to mobile for vaccines. Page placeholders might make this easier.

## Describe the solution you'd like

Placeholders for:
- Sensor/fridge detail page
- New Sensor Page
- Settings Page
- 

## Implementation

Create placeholder pages with navigation

## Describe alternatives you've considered

N/A

## Additional context

N/a

### PR DESCRIPTION
**Pull request by joshxg**
Fixes #3341 

## Change summary

- Simple placeholders for the vaccine module structure

## Testing

Internal

### Related areas to think about

N/A

### CODE REVIEW

File: src/navigation/actions.js [hunk 1]
```diff
@@ -544,3 +544,35 @@ export const updateStocktake = (stocktake, itemIds, name = '') => dispatch => {
 
   dispatch(gotoStocktakeEditPage(stocktake));
 };
+
+export const gotoNewVaccineModulePage = () =>
+  NavigationActions.navigate({
+    routeName: ROUTES.VACCINES_NEW,
+    params: {
+      title: navStrings.current_stock,
```
Comment (mark-prins) [inline src/navigation/actions.js:552]: titles will come later, with implementation, right?

File: src/pages/SettingsPage.js [hunk 2]
```diff
@@ -135,6 +140,7 @@ const Settings = ({ toRealmExplorer, currentUserPasswordHash, requestStorageWrit
         <View>
           <MenuButton text={buttonStrings.realm_explorer} onPress={toRealmExplorer} />
           <MenuButton text={buttonStrings.export_data} onPress={requestStorageWritePermission} />
+          <MenuButton text="New vaccine module" onPress={toNewVaccineModulePage} />
```
Comment (mark-prins) [inline src/pages/SettingsPage.js:143]: and `buttonStrings` is coming later too?

**Review by mark-prins — APPROVED** [review]
Looks good!
Wasn't too sure what the 'new' meant in the NewVaccineModulePage - is it for adding a new 'vaccine module' or a new version of the page? I see that there's a VaccinePage, so assumed that this is a new version and will eventually replace the current one and be renamed. Is that right?

**Comment by joshxg** [comment]
your understanding is what I was going for - sorry for the confusion!

**Pull request merged**